### DISCUSSION TITLE
ESP32: how to use String classes with the buffer

### DISCUSSION THREAD

**Discussion by digamesystems** [rlogiacco/CircularBuffer#55] [Show and tell]
Hi Roberto,

Thanks for the nifty library! I've been using it in an application and today I knocked out an example of how to use it with dynamically allocated String classes on the ESP32. 

I'm offering it up, hoping it might be useful to others. Feel free to improve / modify as desired. 

Regards, 

John
https://github.com/digamesystems/CircularBufferTest/blob/main/src/circularBufferTest/circularBufferTest.ino

**Comment by rlogiacco** [comment]
Thanks a million, very useful!
